.github/workflows/update-semgrep-rules.yml

```diff
@@ -0,0 +1,59 @@
+name: Update Semgrep Rules
+on:
+  workflow_dispatch: {}
+  # TODO: Remove this trigger
+  push:
+    branches:
+      - "cron"
+  #TODO: at some point also set up an actual cron like
+  #  schedule:
+  #  - cron: "43 20 * * *"
+
+jobs:
+  update-semgrep-rules-submodule:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get JWT for semgrep-ci GitHub App
+        id: jwt
+        uses: docker://public.ecr.aws/y9k7q4m1/devops/cicd:latest
+        env:
+          EXPIRATION: 600 # seconds
```

Comment:
why do we need this EXPIRATION?

Reply:
We don't want credentials to have a longer life than they need - this is the shortest setting. Ensure that if an attacker got a hold of these credentials after the job runs, they're expired.